tests/request_helper_test.py

```diff
@@ -103,11 +103,20 @@ def test_is_tracing_returns_what_tracing_percent_method_returns_for_rest(
 
 
 def test_get_trace_id_returns_header_value_if_present(dummy_request):
-    dummy_request.headers = {'X-B3-TraceId': '17133d482ba4f605'}
+    dummy_request.headers = {'X-B3-TraceId': '48485a3953bb6124'}
     dummy_request.registry.settings = {
         'zipkin.trace_id_generator': lambda r: '17133d482ba4f605',
     }
-    assert '17133d482ba4f605' == request_helper.get_trace_id(dummy_request)
+    assert '48485a3953bb6124' == request_helper.get_trace_id(dummy_request)
+
```

Comment:
Nit: could you add another blank line here? We enforce this via flake8, and indeed the travis build for this branch has failed. I believe `make test` is the "run-all-tests" invocation we generally use. :)